templates/email/quote-expired.hypr

```diff
@@ -0,0 +1,26 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+         <!--- Header --->
+            <div class="mz-curbside-email-header">  
+            <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+            <div class="mz-curbside-header-body">
+                <p><strong>{{ siteContext.generalSettings.websiteName }}</strong></p>
+            </div>
+            </div>
```

Comment:
Why is this using the curbside header for a B2B email? Don't do that. Also we should be putting headers in a shared file and reusing it in the B2B templates. I don't know if we did that for the other B2B emails but we should be.

Also please format this file nicely. Proper tabbing/spacing, etc.

This applies to the other emails as well. Please fix all of them.

Reply:
Done, make another css class 'mz-main-email-header' and using for all templates

Reply:
Remove 'mz-main-email-header' and using 'mz-b2b-email-header' class for template